shared/dataflow/codeql/dataflow/DataFlow.qll

```diff
@@ -30,9 +33,27 @@ signature module InputSig {
     ReturnKind getKind();
   }
 
+  /**
+   * A node in the data flow graph that represents an output.
+   */
   class OutNode extends Node;
 
+  /**
+   * A node in the data flow graph representing the value of an argument after
+   * a function call returns. For example, consider the following C++ code:
+   * ```
+   * int a = 1;
+   * increment(&a);
+   * ```
+   * The post-update node for `&a` represents the value of `&a` after
+   * modification by the call to `increment`.
+   */
   class PostUpdateNode extends Node {
+    /**
+     * Gets the node that represents the same value prior to the operation.
+     *
+     * @return The pre-update node.
```

Comment:
I haven't seen this `@return` syntax before. Looks like it might get picked up and rendered nicely by an IDE perhaps.  Is it correct?

Reply:
Maybe Copilot picked that up from [Javadoc](https://www.oracle.com/de/technical-resources/articles/java/javadoc-tool.html#@return)?

Reply:
That seems quite plausible, yeah.  Since I am not sure it's valid syntax, I have removed them and referenced the parameters in the explanation directly (where appropriate).